frontend/src/components/MenuAuth.js

```diff
@@ -1,15 +1,61 @@
 import React from "react";
 import PropTypes from "prop-types";
 import Button from "@material-ui/core/Button";
+import Menu from "@material-ui/core/Menu";
+import MenuItem from "@material-ui/core/MenuItem";
 
-const MenuAuth = ({ userName }) => <Button color="inherit">{userName}</Button>;
+const MenuAuth = ({ onLogout, userName }) => {
+  const [anchorEl, setAnchorEl] = React.useState(null);
+
+  const handleMenuClose = () => {
+    setAnchorEl(null);
+  };
+
+  return (
+    <div>
```

Comment:
I think we can use `Fragment` or `<>` instead of `div`, don't you think?

Reply:
great idea, done!